frontend/packages/ux-editor/src/utils/render.tsx

```diff
@@ -87,29 +87,30 @@ export function renderSelectDataModelBinding({
         selectedElement={dataModelBinding[key]}
         onDataModelChange={onDMChange}
         language={language}
-        noOptionsMessage={t('general.no_options')}
+        noOptionsMessage={(): string => getNoOptionsMessage(t)}
       />
     </div>
   );
-}
+};
 
-export function renderSelectGroupDataModelBinding(
+export const renderSelectGroupDataModelBinding = (
   dataModelBinding: IDataModelBindings,
   onDataModelChange: any,
   language: any,
```

Comment:
Should probably use the `useText`-hook.

Reply:
Sure, I totally agree with you! I updated all places where the language prop was used within the files I touched due to the bugfix. :)